app/static/admin/js/bootstrap.bundle.min.js

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
same note here, please use CDN to load bootstrap js files

Reply:
I'll update it. Thanks